tests/run/i24420-inline-local-ref.scala

```diff
@@ -0,0 +1,12 @@
+inline def f(): Long =
+  1L
+
+inline def g(): Long =
+  inline val x = f()
+  x
+
+inline def h(): Long =
+  inline if g() > 0L then 1L else 0L
+
+@main def Test: Unit =
+  println(h())
```

Comment:
`assert(h() == 1L)` saves a check file. The test rig doesn't provide junit for `assertEquals` IIRC.

Reply:
Good point, will do!